client/lib/feature-detection/index.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+const featureDetection = {
+	/**
+	 * Detects if CSS custom properties are supported
+	 *
+	 * @return {boolean} true when feature is supported
+	 */
+	supportsCssCustomProperties: () => {
```

Comment:
Should we [`memoize`](https://lodash.com/docs/latest#memoize) these or are the underlying checks quick enough to do repeatedly?

Reply:
That'd be very neat! I'd be happy to see that in another PR but would award 12 bonus points for it here :P